src/OpenTelemetry/BaseExportProcessor.cs

```diff
@@ -29,7 +29,9 @@ public enum ExportProcessorType
 /// Implements processor that exports telemetry objects.
 /// </summary>
 /// <typeparam name="T">The type of telemetry object to be exported.</typeparam>
+#pragma warning disable CA1708 // Identifiers should differ by more than case
 public abstract class BaseExportProcessor<T> : BaseProcessor<T>
+#pragma warning restore CA1708 // Identifiers should differ by more than case
```

Comment:
This doesn't seem like a helpful warning to me. I would be in favor of turning CA1708 off for the repo.

Reply:
IMO it is problematic only for protected fields, these are usually not the best possible options.